example/multi-package/entrypoint.sh

```diff
@@ -2,13 +2,29 @@
 
 # this file is used by .github/workflows/integration-test.yml
 
+if [[ "$(which gripmock)" == "" ]]; then
```

Comment:
please add comment explaining the branching logic

Reply:
updated